app/views/posts/index.html.erb

```diff
@@ -1 +1,30 @@
-<h1>Users Posts</h1>
\ No newline at end of file
+<section class="d-flex align-items-center">
+  <%= image_tag @user.photo, :class => 'user-photo', alt: 'User photo' %>
+  <div class="pl-2 flex-grow-1 d-flex justify-content-around align-items-center">
+    <h2 class="fs-2"><%= @user.name %></h2>
+    <p>Number of posts: <%= @user.posts_counter%></p>
+  </div>
+</section>
+<section>
+  <% @posts.each do |post| %>
+    <%= link_to user_post_path(@id, post.id), class: "user-link" do %>
+      <div class="container-fluid border">
+        <h3><%= post.title %></h3>
+        <p><%= post.text %></p>
+        <div class="d-flex justify-content-end">
+          <p>Comments: <%= post.comments_counter.nil? ? 0 : post.comments_counter %></p>
+          <p class="mx-2">Likes: <%= post.likes_counter.nil? ? 0 : post.likes_counter %></p>
+        </div>
+      </div>
+    <% end %>
+    <div class="container-fluid border">
+      <% @comments = Post.find(post.id).comments %>
+        <% @comments.each do |comment| %> 
+          <div class="d-flex align-items-center">
+            <h3><%= comment.user.name %></h3>:
+            <h5 class="mx-2"><%= comment.text %></h5> 
+          </div>   
+        <% end %>
```

Comment:
Likewise, show the 5 most recent comments on the post here.

![image](https://user-images.githubusercontent.com/27709832/230358163-f2d5052e-7a59-4d82-b188-b96bb042e8a9.png)

**Hint**

```
post.recent_comments.each |do|
  // comment info
end
```

The above block point to the method below defined in the Post model.

https://github.com/leehaney254/blog-app/blob/f1a7aeb10f6668913686fd6bc02f2efb66b0e17e/app/models/post.rb#L11-L13

Reply:
Thank you. I have fixed it.